packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -179,6 +186,13 @@ const Sublayout = ({ children, isInsideCommunity }: SublayoutProps) => {
             resizing,
           )}
         >
+          {eventMarqueeEnabled && routesWithEventMarquee && (
+            <EventMarquee
+              isSticky={true}
+              elementId="event-marquee-sticky"
+              zIndex={25}
```

Comment:
we dont use this comp anywhere else so these props could be default or evvent we dont need them to be props, just values in the component